packages/language-support/src/autocompletion/completionCoreCompletions.ts

```diff
@@ -702,7 +716,11 @@ function completeAliasName({
     candidateRule.ruleList.includes(CypherParser.RULE_createAlias) &&
     candidateRule.ruleList.includes(CypherParser.RULE_aliasName)
   ) {
-    return parameterSuggestions;
+    return parameterSuggestions.map((completionItem) => ({
+      label: completionItem.label,
+      kind: completionItem.kind,
+      insertText: backtickDbNameIfNeeded(completionItem.label),
```

Comment:
Can a parameter include the `.`, otherwise this should be `backtickIfNeeded`, no?

Reply:
Ah yeah, this is not valid